content/developer/Theme/Icons/SuitePIcons/_index.en.adoc

```diff
@@ -0,0 +1,9 @@
+---
+title: "SuiteP Icons"
+weight: 20
+pre: "<b> </b>"
+---
+
+=== Table of Contents
```

Comment:
I think there are too many sub-page levels here, with tiny pages and too many pages with nothing but a table of contents. I would suggest pulling all three sub pages into this one, removing the Table of contents. This will also de-clutter the menu in the sidebar.
So, this page page `SuiteP Icons` simply includes headers (not sub-pages) for Accordion, Admin and Sidebar icons. 
I also suggest adding a couple of lines in the beginning as an introduction. Like `You can use these icons for your custom modules`, or `here is a reference of the icons currently used`, or whatever is appropriate.